### DISCUSSION TITLE
🎧 Kaizen 15 - NOT a pipe dream- Jun. 28, 2024

### DISCUSSION THREAD

**Discussion by gerhard** [thechangelog/changelog.com#511] [Kaizen]
![shields.io](https://img.shields.io/badge/Last%20updated%20on%20August%2015%2C%202024-PUBLISHED-blue?style=for-the-badge)

[🎧 This is now available as changelog.com/friends/50](https://changelog.com/friends/50)

<a href="https://changelog.com/friends/50" target="_blank"><img width="1549" alt="image" src="https://github.com/user-attachments/assets/0b7f5e0c-0c86-4ca5-ab41-49e0ada0410d"></a>

See all improvements in this 🐙 GitHub repository: [Kaizen 14 ... 15](https://github.com/thechangelog/changelog.com/compare/kaizen.14...kaizen.15)

---

This contains all things related to the next Kaizen episode.

Share your thoughts via comments below:
- Is there something in changelog.com that you, our dear listener, would like us to tackle?
- Is there something that **you** would like to help out with?
- Anything that we - @adamstac @jerodsanto - want to improve?

A rough timeline of what to expect (red line is today):

```mermaid
gantt
    title 🎧 Kaizen 15 timeline
    dateFormat  YYYY-MM-DD
    axisFormat %b %e, %Y

    section CONTRIBUTE
    💻 ALL CONTRIBUTIONS WELCOME :2024-04-05, 77d

    section RECORD

    🎙 Record :milestone, 2024-06-21
    ✂️ Edit :2024-06-21, 7d
    🚀 SHIP IT! :milestone, 2024-06-28

    section NEXT
    Kick-off Kaizen 16 :2024-06-28, 30d
```

---

### What changed? [Kaizen.14 ... Kaizen.15](https://github.com/thechangelog/changelog.com/compare/kaizen.14...master)

- #510
- #513 🙏 @br3ndonland 
- #515
- #516
- #517
- #518

---

### What else is coming up?

- Daggerize `action-slack-notify`
  - https://github.com/gerhard/daggerverse/tree/main/notify
- The Dagger Factory 🏭
  - [Functional CI](https://daggerverse.dev)
  - New UI 😱

---

### Links for the show notes

- https://en.wikipedia.org/wiki/Navel_gazing
- https://elixir-lang.org/blog/2024/06/12/elixir-v1-17-0-released/
  - https://news.ycombinator.com/item?id=40656747
- https://www.postgresql.org/docs/current/sql-vacuum.htm
- https://github.com/inception-health/otel-export-trace-action/issues/194
  - https://github.com/corentinmusard/otel-cicd-action
- https://fly.io/infra-log/
- https://fly.io/phoenix-files/world-page-speed-test-elastic-scale-with-flame/
- https://worldpagespeed.fly.dev/
- https://pipedream.changelog.com
  - [🎬 Let's build a CDN](https://www.youtube.com/watch?v=8bDgWvyglno)

---

### This is a follow-up to
- https://github.com/thechangelog/changelog.com/discussions/499
- https://github.com/thechangelog/changelog.com/discussions/485
- https://github.com/thechangelog/changelog.com/discussions/480

**Comment by jerodsanto** [comment]
In addition to the News redesign, I'd like to talk about a few improvements I made between Kaizens:

- Adding episode flavors to 'The Changelog' feed
- Podlove chapters for Spotify
- Custom feeds feature, Stripe & ++ plan
- Dynamic news/episode share images

**Comment by jerodsanto** [comment]
It's alive! https://changelog.com/friends/50
